src/components/Answer/Answer.js

```diff
@@ -12,7 +13,11 @@ export default class Answer {
 
   getScore() {
     if (this.checkIfCorrect()) {
-      return 1;
+      if (this.changed) {
+        return 0.5;
+      } else {
+        return 1;
+      }
     } else {
       return 0;
```

Comment:
Soon this method should be refactored. Maybe do it now, before it gets too big:
Let's assume that the default score is 0,
in 1st if statement check if it is correct - assign 1; 
in  2nd if check if it is changed - divide current score it by 2;

and simply return score variable;